sourcegraph/cody
# Cleanup chat model selector styles

This cleans up the chat model selector styles so it now has a keyboard focus outline, fixes some bugs with changing model with the keyboard, more closely matches VS Code styles (the webui toolkit is out of date), leaves space on the right for other buttons in the future (e.g. raw view), and includes the chat provider icon at the top of the chat.

| Before | After |
|-|-|
| <img width="1372" alt="Screenshot 2023-11-15 at 5 56 46 pm" src="https://github.com/sourcegraph/cody/assets/153/340d7b63-41e5-44b4-af4d-293984d0f0f3"> | <img width="1372" alt="Screenshot 2023-11-15 at 5 55 27 pm" src="https://github.com/sourcegraph/cody/assets/153/77d29e09-4030-424f-be64-5bd07a82a061"> |

## Test plan

- Start a new chat, select items
- Open an existing chat, verify disabled state
